### PR TITLE
16 utilize vercel image optimizations

### PR DESCRIPTION
**Pull request by kobebuckley**
Update representing the Image implementation suggestions from Next js for optimization as well as required alt tag additions with relevant descriptions

### CODE REVIEW

File: src/components/HomePage.js [hunk 1]
```diff
@@ -8,7 +9,7 @@ const HomePage = () => {
       <NavBar />
       <div className="max-h-full flex items-center justify-center flex-col lg:flex-row">
         <div className="scale-[80%]  2xl:scale-90 xl:pl-20 flex flex-col items-center">
-          <img src="hero-image.png" className="w-full max-w-[900px] animate-float" />
+          <Image alt='This is a picture of an island with palm trees, a pineapple, coconut, surfboard, flower, starfish, shells, river, sandals, and a laptop.' src="hero-image.png" className="w-full max-w-[900px] animate-float" />
```
Comment (MichaelTamaki) [inline src/components/HomePage.js:12]: Nice work! For the alt tags you don't need to be so verbose. And for images that are purely decorative like this hero image, you can have blank alt text. 

```suggestion
          <Image alt='' src="hero-image.png" className="w-full max-w-[900px] animate-float" />
```

See some examples:

https://www.w3.org/WAI/WCAG21/Techniques/html/H37

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attributes

File: src/components/Logo.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 
 
 
 const Logo = () => {
   return (
     
       <Link href="/" whileHover={{ scale: 1.5 }}>
-        <img
+        <Image
+          alt='This is a logo of Code with Aloha represented by the name, as well as an orange lightbulb with the torch like yellow flame akin to a sunset scene.'
```
Comment (MichaelTamaki) [inline src/components/Logo.js:12]: Example of reducing verbosity:
```suggestion
          alt='Code with Aloha'
```

This ensures that it serves the same functionality as it does for sighted users. The visual decorations aren't important to the functionality of the image.

**Review by MichaelTamaki — APPROVED** [review]
LGTM! Have suggestions but they are not blockers 💯

**Review by Alexandra-Haynes — CHANGES_REQUESTED** [review]
@kobebuckley , thank you for this PR 🔥 I think transitioning to the Image component has some strong advantages over the standard  tag.

I've noticed a couple of areas we could improve with the <Image> component usage:

**1. Alt Text**: I agree with @MichaelTamaki  that the alt description for the images are quite detailed, which is great, but it might be a bit lengthy for accessibility users. Consider simplifying it for clarity. Here's a suggestion: alt="_Code with Aloha logo featuring an orange lightbulb_." or simply alt="_Code with Aloha logo_" Keeping the alt description short provides efficiency for screen readers that vocalize the alt text and it also quickly provides the context and relevance of the image, without overwhelming the user. 

**2. Width & Height Properties:**

For the next/image component, it's essential to specify the intrinsic dimensions of the image. Without specifying them, we get a server error: _Error: Image with src "../logo.png" is missing required "width" property_. For these properties we have to look at the natural width and heights of the pics (by looking at the properties), this will assure the natural ratio. The actual rendered dimensions will be controlled by tailwind classes.  This means the image won't look stretched or squished. 
That being said, for the logo dimensions are **500px by 500px** and for the hero_image: **1300px by 1300px**. Please add these dimensions as width and height attributes.

I hope this feedback helps in refining the code. Let me know your thoughts!

**Comment by kobebuckley** [comment]
> LGTM! Have suggestions but they are not blockers 💯

Thank you for providing examples, links, and suggestions for fixing up the code as shown below. I will implement these changes moving forward

**Comment by kobebuckley** [comment]
Thank you Alexandra; I'm still working on the 2nd one, but I'll get it fixed

**Comment by Alexandra-Haynes** [comment]
Kobe, thanks for working on this! I was reviewing your pull request and noticed the Image component usage in your code.


In Next.js, the width and height props of the Image component should be written with curly braces instead of quotation marks. See example here: https://nextjs.org/docs/pages/api-reference/components/image#width This syntax is used because the values for these props are expected to be numbers, not strings.

Would you mind updating the width and height to use curly braces like this? width={500} height={500}


This change will ensure that the props are passed as numbers and help to avoid any potential issues related to prop types.

Thanks again for your work, and let me know if you have any questions!

**Comment by kobebuckley** [comment]
> Kobe, thanks for working on this! I was reviewing your pull request and noticed the Image component usage in your code.
> 
> In Next.js, the width and height props of the Image component should be written with curly braces instead of quotation marks. See example here: https://nextjs.org/docs/pages/api-reference/components/image#width This syntax is used because the values for these props are expected to be numbers, not strings.
> 
> Would you mind updating the width and height to use curly braces like this? width={500} height={500}
> 
> This change will ensure that the props are passed as numbers and help to avoid any potential issues related to prop types.
> 
> Thanks again for your work, and let me know if you have any questions!

Thank you Alexandra! I replaced the string values with the proper prop types for numbers instead to represent Next js syntax

**Review by Alexandra-Haynes — APPROVED** [review]
Thank you, Kobe! We appreciate your help

**Pull request merged**